app/views/users/me.html.erb

```diff
@@ -3,11 +3,21 @@
   <%= javascript_include_tag 'blueimp_script' %>
 <% end %>
 
-<%= render 'camps/dont_miss_out', except: nil %>
-<div class="me col-md-5 col-xs-12" dir="<%=t :html_direction %>" style="float: <%=t :lang_direction%>">
+<%
+=begin%>
+ <%= render 'camps/dont_miss_out', except: nil %> 
+<%
+=end%>
```

Comment:
Looks like there are some strange linebreaks in the code here, maybe a prettify-error?

Reply:
Fixed!